airflow/ti_deps/deps/ready_to_reschedule.py

```diff
@@ -40,7 +40,8 @@ def _get_dep_statuses(self, ti, session, dep_context):
         considered as passed. This dependency fails if the latest reschedule
         request's reschedule date is still in future.
         """
-        if not getattr(ti.task, "reschedule", False):
+        is_mapped = ti.task.is_mapped
+        if not is_mapped and not getattr(ti.task, "reschedule", False):
```

Comment:
Worth a comment here (and below) to note why we explicitly exclude mapped operators this this function.